src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.Windows.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 #if FEATURE_APPX
+#nullable enable
```

Comment:
This should probably be outside of the #if def.

Reply:
your tool did it :P